ovos_local_backend/configuration.py

```diff
@@ -35,28 +35,15 @@
     "admin_key": "",  # To enable simply set this string to something
     "skip_auth": False,  # you almost certainly do not want this, only for atypical use cases such as ovos-qubes
     "default_location": {
-        "city": {
-            "code": "Lawrence",
-            "name": "Lawrence",
-            "state": {
-                "code": "KS",
-                "name": "Kansas",
-                "country": {
-                    "code": "US",
-                    "name": "United States"
-                }
-            }
-        },
-        "coordinate": {
-            "latitude": 38.971669,
-            "longitude": -95.23525
-        },
-        "timezone": {
-            "code": "America/Chicago",
-            "name": "Central Standard Time",
-            "dstOffset": 3600000,
-            "offset": -21600000
-        }
+        "city": "Lawrence",
+        "address": "Lawrence",
```

Comment:
I think this should stay "name" instead of "address"
I use the "name" field in my weather, because the best weather station is actually in a different "city", or actually it is classified as a "village", but the actual "city" I live in is different.

An address should be there for getting directions home, or asking what my address is, or something like that.  The name is just for display purposes and I am one that does use it

Reply:
I don't really want to change this field here in config, the aim was to have a single dict format wherever location is used, unfortunately the dict above is used in a lot of places, lets keep using the same format all over ovos ecosystem

Reply:
Just trying to get a logical consistence with what geolocation services can provide/are providing regarding your _**current location**_. Nested or flat doesn't matter (and everything externally requesting the location is served nested btw). Internally we should use a flat dict, easier to handle, ie kwargs.

If @builderjer wants to use it as weather location that is not his actual location, ok. But it will be in the database as such (if name or adress or whatever). We should separate weather and location, since those can be and appearently are used in different ways.

Reply:
I just change the visual name of my location. And I that is directly connected to what the weather skill displays. It still uses the actual location for fetching weather. I am probably the one exception that uses it this way, as usual, but when you live in rural areas, weather stations are spotty. I am so rural that "Verizon can you hear me now" answers "what?".  Ideally, and I know it's on the road map, would be to use another provider that has a station near by. And every provider will return a different json, so catching them is the issue.

Reply:
This is what my location looks like
```
  "location": {
    "city": {
      "code": "Orchard City, Delta County, Colorado, 81418, United States",
      "name": "Hotchkiss",
      "state": {
        "code": "Colorado",
        "country": {
          "code": "United States",
          "name": "US"
        },
        "name": "CO"
      }
    },
    "coordinate": {
      "latitude": 38.8283161,
      "longitude": -107.9708995
    },
    "timezone": {
      "code": "America/Denver",
      "dst_offset": 3600000,
      "name": null,
      "offset": -21600000
    }
  },
```
It picks up weather from 'orchard city' as that is the best station for my house, but my address is Hotchkiss, so this way, it shows Hotchkiss as my location.  Even with "hey mycroft, what is my location".  Like I said, I am probably the only one that uses it this way